src/main/java/gitflow/actions/RepoActions.java

```diff
@@ -105,13 +104,13 @@
                 actionList.add(new FinishReleaseAction(myRepo));
 
                 //can't publish release if it's already published
-                if (branchUtil.isCurrentBranchPublished()==false){
+                if (!branchUtil.isCurrentBranchPublished()){
                     actionList.add(new PublishReleaseAction(myRepo));
                 }
             }
 
             //make sure there's something to track and that not all features are tracked
-            if (noRemoteTrackBranches==false  && trackedAllReleaseBranches ==false){
+            if (!noRemoteTrackBranches && !trackedAllReleaseBranches){
```

Comment:
Please retain the existing conditional formatting. It's more verbose for readability purposes